concepts/05 UI Components/List/00 Getting Started with List/03 Bind the List to Data.md

```diff
@@ -0,0 +1,182 @@
+The **List** can load data from different data source types. To use a local array, assign it to the [dataSource](Documentation/ApiReference/UI_Components/dxList/Configuration/#dataSource) property and specify the name of the data field you would like to display in the [displayExpr](/Documentation/ApiReference/UI_Components/dxList/Configuration/#displayExpr) property. For information on data binding with other data source types, refer to the following articles:
```

Comment:
> specify the name of the data field you would like to display in the [displayExpr](/Documentation/ApiReference/UI_Components/dxList/Configuration/#displayExpr) property

We need to eliminate a couple of ambiguities here: we display the field values, not the name nor the data field itself, and we display them in the List, not in the displayExpr property.

```suggestion
The **List** can load data from different data source types. To use a local array, assign it to the [dataSource](Documentation/ApiReference/UI_Components/dxList/Configuration/#dataSource) property and use the [displayExpr](/Documentation/ApiReference/UI_Components/dxList/Configuration/#displayExpr) property to specify the name of the data field whose values you would like to display. For information on data binding with other data source types, refer to the following articles:
```